### ISSUE TITLE
Hardi Rubicon 9000 - incorrect positioning, and crazy driving ;)

### ISSUE THREAD

**Issue by S-eler** [Stephan-S/FS22_AutoDrive#825]
Hi!

Autodrive (or base game, I'm not sure which side is responsible) incorrectly "read" front and end of the vehicle - only the main chassis, not counting booms.

This is a problem for reversing (is trying to drive back when is already blocked by booms to the wall). It's also visible when it's parked. Most of vehicles drive as far as front bumper, like on photo below:
![IMG_0467](https://user-images.githubusercontent.com/121870418/218452149-1391e122-0be0-4d2f-a9c2-1ee28352829f.jpeg)

But Rubicon drive even further than front axle (you see parking point under the vehicle):
![IMG_0468](https://user-images.githubusercontent.com/121870418/218452266-d9d1340c-ab6a-4851-bbbc-aa87fc75338e.jpeg)

The fact that it's locating itself behind turning axle provide really crazy driving by AI - it's go of the road on almost every turn, I was forced to rebuild all my network. Driving by myself, it's got very small turning radius - driving by auto drive, it's even worse than semi-trailer with the biggest semi-trailer (tlx x52 tipper).

**Comment by Iwan1803** [comment]
It's because of the model and we can't change this.
If we were to define any exemption rules for vehicles just to make them park properly for you, that would not be helpful.

**Comment by Axel32019** [comment]
no feedback -> closed
